boot/bootutil/include/bootutil/bootutil_log.h

```diff
@@ -74,34 +74,58 @@ extern "C" {
 #define BOOT_LOG_LEVEL_INFO	3
 #define BOOT_LOG_LEVEL_DEBUG	4
 
+/*
+ * The compiled log level determines the minimum level that can be
+ * printed.  Messages at or below this level can be printed, provided
+ * they are also enabled through the Rust logging system, such as by
+ * setting RUST_LOG to bootsim::api=info.
+ */
 #ifndef BOOT_LOG_LEVEL
-#define BOOT_LOG_LEVEL BOOT_LOG_LEVEL_ERROR
+#define BOOT_LOG_LEVEL BOOT_LOG_LEVEL_INFO
```

Comment:
Maybe then the patch to change to info comes first, so the next patch is useful out of the box?

Again, though, I'm not trying to stop the merge with this comment.